src/app/modules/components/aggregate-data/aggregate-data.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="aggregate-results">
+  <div class="aggregate">
+    <div fxLayout="row" fxLayoutAlign="space-between" class="mobile">
+      <div class="territory-name">
+        <div>
+          <h2 class="title">
+            Diários Oficiais agregados de 
+            {{name}}
+          </h2>
+          <p>A tabela apresenta informações sobre o arquivo XML a ser baixado.</p>
```

Comment:
Como comentei antes, aqui seria o ZIP então acho que faz mais sentido omitir o tipo do arquivo mesmo.
```suggestion
          <p>A tabela apresenta informações sobre o arquivo a ser baixado.</p>
```